apps/internal/local/server.go

```diff
@@ -84,7 +85,7 @@ func New(reqState string, port int) (*Server, error) {
 
 	serv := &Server{
 		Addr:     fmt.Sprintf("http://localhost:%s", portStr),
-		s:        &http.Server{Addr: "localhost:0"},
+		s:        &http.Server{Addr: "localhost:0", ReadHeaderTimeout: time.Second},
```

Comment:
This is the [Slowloris](https://en.wikipedia.org/wiki/Slowloris_(computer_security)) fix. We could simply disable the check because there's no vulnerability here--this is the only server in MSAL and it runs on localhost. If users want to Slowloris the local server trying to help them authenticate, that's okay. But the header timeout won't hurt anything and it feels cleaner to me than disabling a warning :shrug: